internal/agent/device/status/containers.go

```diff
@@ -63,53 +47,8 @@ type Shell interface {
 	Command(cmd string) (output []byte, err error)
 }
 
-func podmanApplicationStatus(entry PodmanContainerListEntry) v1alpha1.ApplicationStatusType {
-	switch entry.State {
-	case "running":
-		return v1alpha1.ApplicationStatusRunning
-	case "exited":
-		if entry.ExitCode == 0 {
-			return v1alpha1.ApplicationStatusCompleted
-		}
-		return v1alpha1.ApplicationStatusError
-	default:
-		return v1alpha1.ApplicationStatusUnknown
-	}
-}
-
-func (c *Container) PodmanExport(ctx context.Context, status *v1alpha1.DeviceStatus) error {
-	podmanExecCtx, cancel := context.WithTimeout(ctx, podmanCommandTimeout)
-	defer cancel()
-	args := []string{"ps", "-a", "--format", "json"}
-	for _, pattern := range c.matchPatterns {
-		args = append(args, "--filter")
-		args = append(args, fmt.Sprintf("name=%s", pattern))
-	}
-	podmanOut, podmanErrOut, podmanExitCode := c.exec.ExecuteWithContext(podmanExecCtx, podmanCommand, args...)
```

Comment:
e2e  and integration fails because we reported status of container images via podman such as flightctl-db this needs a different and more reasonable signal.